clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/pvc.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
 spec:
   accessModes:
     - ReadWriteOnce
+  {{ if .UseCSI }}
+  storageClassName: csi-gce-pd
+  {{ end }}
```

Comment:
Just out of curiosity - what will it do with UseCSI is false?

Reply:
that storage class line is just not added, so the PVC will be identical to the current non csi one.